 * Serving Flask app "index" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "manage.py", line 20, in <module>
    manager.run()
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 417, in run
    result = self.handle(argv[0], argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 386, in handle
    res = handle(*args, **config)
  File "/usr/local/lib/python2.7/dist-packages/flask_script/commands.py", line 479, in __call__
    **self.server_options)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 943, in run
    run_simple(host, port, self, **options)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 795, in run_simple
    s.bind(get_sockaddr(hostname, port, address_family))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
 * Serving Flask app "index" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
 * Running on http://0.0.0.0:8080/ (Press CTRL+C to quit)
 * Restarting with stat
 * Debugger is active!
 * Debugger PIN: 766-714-746
 * Serving Flask app "index" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    manager.run()
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 417, in run
    result = self.handle(argv[0], argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 386, in handle
    res = handle(*args, **config)
  File "/usr/local/lib/python2.7/dist-packages/flask_script/commands.py", line 479, in __call__
    **self.server_options)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 943, in run
    run_simple(host, port, self, **options)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 795, in run_simple
    s.bind(get_sockaddr(hostname, port, address_family))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
 * Serving Flask app "index" (lazy loading)
 * Environment: production
   WARNING: Do not use the development server in a production environment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "manage.py", line 21, in <module>
    manager.run()
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 417, in run
    result = self.handle(argv[0], argv[1:])
  File "/usr/local/lib/python2.7/dist-packages/flask_script/__init__.py", line 386, in handle
    res = handle(*args, **config)
  File "/usr/local/lib/python2.7/dist-packages/flask_script/commands.py", line 479, in __call__
    **self.server_options)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 943, in run
    run_simple(host, port, self, **options)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 795, in run_simple
    s.bind(get_sockaddr(hostname, port, address_family))
  File "/usr/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
172.17.3.90 - - [13/Nov/2018 04:56:07] "GET /register HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:08] "GET /6df5546dfecb3262337cb56455b9134c.svg HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:08] "GET /api/users/undefined HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:10] "GET /register HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:10] "GET /6df5546dfecb3262337cb56455b9134c.svg HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:10] "GET /api/users/undefined HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:11] "GET /register HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:11] "GET /6df5546dfecb3262337cb56455b9134c.svg HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 04:56:11] "GET /api/users/undefined HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:16] "GET / HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:16] "GET /dist/runtime~main.b6ad9ed301cc83f945fd.js HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /dist/vendor.4e09edc8e94bec598c3c.chunk.js HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /dist/main.b8b7a9e88480cdff8928.chunk.js HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /api/users/undefined HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /6df5546dfecb3262337cb56455b9134c.svg HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /dist/favicon.ico/ HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:17] "GET /dist/icon_180x180.33285f999db78beed5640085c051e7b1.png/ HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:28] "POST /api/get_token HTTP/1.1" 403 -
172.17.1.53 - - [13/Nov/2018 11:24:46] "POST /api/create_user HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:47] "GET /api/entries HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:24:47] "GET /api/users/active HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:25:26] "POST /api/create_offer HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:25:26] "GET /api/entries HTTP/1.1" 200 -
172.17.1.53 - - [13/Nov/2018 11:25:26] "GET /api/users/active HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:22] "GET / HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:22] "GET /dist/runtime~main.b6ad9ed301cc83f945fd.js HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:23] "GET /dist/vendor.4e09edc8e94bec598c3c.chunk.js HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:23] "GET /dist/main.b8b7a9e88480cdff8928.chunk.js HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:24] "GET /api/users/undefined HTTP/1.1" 200 -
172.17.3.90 - - [13/Nov/2018 16:00:24] "GET /6df5546dfecb3262337cb56455b9134c.svg HTTP/1.1" 200 -
 * Detected change in '/home/fazatholomew/Documents/sis/application/app.py', reloading
[<User 1>, <User 2>, <User 3>, <User 4>, <User 5>]
[<User 1>, <User 2>, <User 3>, <User 4>, <User 5>]
